docs/use_cases/model_applications/convection_allowing_models/README.rst

```diff
@@ -1,2 +1,4 @@
 Convection Allowing Models
 --------------------------
+
+ High resolution model configurations (1-4km) usually producing forecasts between 0-3 days.  Also referred to as limited area models, stand-along regional, and short range weather applications
```

Comment:
High resolution model configurations (1-4km) usually producing forecasts between 0-3 days (also referred to as limited area models, stand-alone regional, and short range weather applications).